OPEA-Comps/tests/test_app.py

```
@@ -0,0 +1,43 @@
+import pytest
+from fastapi.testclient import TestClient
+from app import ExampleService
+import aiohttp
+from unittest.mock import Mock, patch
```

Ruff (0.8.2)

`unittest.mock.Mock` imported but unused

Remove unused import: `unittest.mock.Mock`

(F401)